zsign.cpp

```diff
@@ -9,6 +9,7 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <filesystem>
```

Comment:
Using `<filesystem>` header requires C++17; there was, however, a commit by @zhlynn to remove non-C++11 features, s.t. the project can build w/ `-std=c++11`.

I believe, however, that this is the most portable way of achieving such functionality.  What do you think, @zhlynn?